### PR TITLE
Nextcloud: add version 2.5.3

### PR DESCRIPTION
**Pull request by linsui**
Maybe we need something like `-gui` option, so we can always run gui from cmd.

### CODE REVIEW

File: bucket/nextcloud.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "2.5.3",
+    "description": "Desktop sync client for Nextcloud, a self-hosted productivity platform that keeps you in control.",
+    "homepage": "https://nextcloud.com/",
+    "license": "GPL-2.0-or-later",
+    "url": "https://download.nextcloud.com/desktop/releases/Windows/Nextcloud-2.5.3-setup.exe#/dl.7z",
+    "hash": "c75f667e6a149f5f72db46d15b4ea24a614789a2cf103040c2f9a66fc3482466",
+    "bin": [
```
Comment (Ash258) [inline bucket/nextcloud.json]: Keep defaults. You cannot be sure if application is not calling it statically.

```suggestion
    "bin": [
        "nextcloudcmd.exe",
````

File: bucket/nextcloud.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        ]
+    ],
+    "shortcuts": [
+        [
+            "nextcloud.exe",
+            "Nextcloud"
+        ]
+    ],
+    "checkver": {
+        "url": "https://nextcloud.com/install/#install-clients",
```
Comment (Ash258) [inline bucket/nextcloud.json]: Label is only for browsers. Webclients are ignoring them as they cannot be evaluated in most cases

```suggestion
        "url": "https://nextcloud.com/install/",
```

File: bucket/nextcloud.json (hunk 1 above)
Comment (linsui) [reply bucket/nextcloud.json]: How about

```
    "bin": [
        "nextcloudcmd.exe",
        [
            "nextcloudcmd.exe",
            "nextcloud"
        ]
    ],
```

or

```
    "bin": [
        "nextcloud.exe",
        [
            "nextcloudcmd.exe",
            "nextcloud"
        ]
    ],
```

Comment (Ash258) [reply bucket/nextcloud.json]: My suggestion will make it this:

```json
"bin": [
    "cmd",
    [
         "cmd",
         "simple"
    ]
]
```

Both codes you provided are identical and same to what my suggestion is ...

File: bucket/nextcloud.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.5.3",
+    "description": "Desktop sync client for Nextcloud, a self-hosted productivity platform that keeps you in control.",
+    "homepage": "https://nextcloud.com/",
+    "license": "GPL-2.0-or-later",
+    "url": "https://download.nextcloud.com/desktop/releases/Windows/Nextcloud-2.5.3-setup.exe#/dl.7z",
+    "hash": "c75f667e6a149f5f72db46d15b4ea24a614789a2cf103040c2f9a66fc3482466",
```
Comment (Ash258) [inline bucket/nextcloud.json]:
```suggestion
    "hash": "c75f667e6a149f5f72db46d15b4ea24a614789a2cf103040c2f9a66fc3482466",
    "pre_install": "'$R*', '$P*', 'vc_redist*' | ForEach-Object { Remove-Item \"$dir\\$_\" -Force -Recurse }",
```

File: bucket/nextcloud.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.5.3",
+    "description": "Desktop sync client for Nextcloud, a self-hosted productivity platform that keeps you in control.",
+    "homepage": "https://nextcloud.com/",
+    "license": "GPL-2.0-or-later",
```
Comment (Ash258) [inline bucket/nextcloud.json]:
```suggestion
    "license": "GPL-2.0-or-later",
    "suggest": {
        "Microsoft Visual C++ Redistributable 2017": "extras/vcredist2017"
    },
```

**Pull request merged**